eyeshade/workers/reports.js

```diff
@@ -1181,7 +1200,7 @@ exports.workers = {
             provider = entry && entry.provider
             if (provider && entry.parameters) wallet = await runtime.wallet.status(entry)
 
-            if ((!wallet) || (!wallet.address) || (!wallet.defaultCurrency)) {
+            if (!includeUnpayable && (!wallet || !wallet.address || !wallet.defaultCurrency)) {
```

Comment:
it would be nice if we can consolidate this with the reworked logic you did above using `validateWallet`, not critical though